ee/clickhouse/sql/clickhouse.py

```diff
@@ -51,8 +54,20 @@ def kafka_engine(
         return KAFKA_ENGINE.format(topic=topic, kafka_host=kafka_host, group=group, serialization="JSONEachRow")
 
 
-def ttl_period(field: str = "created_at", weeks: int = 3):
-    return "" if settings.TEST else f"TTL toDate({field}) + INTERVAL {weeks} WEEK"
+def ttl_period(field: str = "created_at", value: int = 3, interval_unit: str = "WEEK"):
```

Comment:
We could simplify things if we took an argument like `interval: str = "3 WEEKS"` as an argument - all callsites are currently static and we can drop complexity.